common/faults/fault_config.json

```diff
@@ -199,6 +199,15 @@
                     "time_to_latch": 10,
                     "time_to_unlatch": 1000,
                     "lcd_message": "APPS Brake Fault"
+                },
+                {
+                    "fault_name": "dual_pedal_act",
```

Comment:
APPS_BRAKE was originally the fault meant to handle the dual pedal fault, so get rid of that fault if you are changing the name. You can also just change the message displayed on the screen for APPS_BRAKE

Reply:
Will address